Python/BitCoin Mining/README.md

```diff
@@ -0,0 +1,35 @@
+# What Is Bitcoin Mining?
+Bitcoin mining is performed by high-powered computers that solve complex computational math problems; these problems are so complex that they cannot be solved by hand and are complicated enough to tax even compelling computers.
+
+# Terminology :
+
+1. Transaction: An transaction is a transfer of Bitcoin value and is collected in Blocks 
+
+2. SHA256: a function that can generate an almost-unique 256-bit (32-byte) signature(Hexa-Decimal) for a text.
+
+3. Block: Blocks are files where data about the Bitcoin network are permanently recorded
+
+4. Nonce: Miners have to guess this number which will reward them BTC. A nonce is an abbreviation for "number only used once," which is a number added to a hashed—or encrypted—block in a blockchain that, when rehashed, meets the difficulty level restrictions. 
+
+5. Difficulty Level: Number of prefix Contagious Zeroes 
+
+# Theory :
+BTC will be rewarded if Nonce gives us a string with Zeroes' prefix the difficulty number of times. Sounds confusing?
+So, Let's take an example :
+
+1. Let's assume we have a difficulty level = 20 
+2. Now, Let's again assume that we have the previous hash as: 'a5x208fecf9a66be9a2bc7...'
+3. Now we create text as  : text = str(block_number) + transactions + previous_hash + str(nonce)
+4. Now, we pass text to SHA256 Function to generate hash
+5. Finally, that hash prefix must be equal to the number of zeroes as difficulty level.
+6. And Boom!! You have mined successfully.
+

```

Comment:
You haven't explained anywhere, how to run the script.
Add test cases/screenshots